runtime/proto/service/corfu_message.proto

```diff
@@ -22,7 +22,8 @@ enum PriorityLevel {
 }
 
 message ProtocolVersionMsg {
-  fixed64 capability_vector = 1;
+  repeated fixed64 capability_vector = 1;
+  int64 corfu_source_code_version = 2;
```

Comment:
maybe a comment explaining how this is populated?

Reply:
Please address this.